### DISCUSSION TITLE
Audio Asset for IOS

### DISCUSSION THREAD

**Discussion by crzaratech** [gdelataillade/alarm#65] [Q&A] [answered]
A while back I started a project with instructions pointing to this [How to import your assets to Xcode](https://gist.github.com/gdelataillade/68834caacdd6727f1418e46788f70b53). Im setting up another project and since the last update I don't see those instructions anymore. 

Just wondering if that is still needed anymore or removed by accident :)

**Comment by gdelataillade** [comment, marked as answer]
Hey @crzaratech !

You don't need to follow those instructions anymore. I found a way to import Flutter assets directly from the native code.

Let me know if you have any problem.
